test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -9,14 +9,16 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         if ($IsWindows) {
             $userName = "testuserservices"
             $testPass = "Secret123!"
+            $SecurityDescriptorSddl = 'D:(A;;CCLCSWRPWPDTLOCRRC;;;SY)(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;BA)(A;;CCLCSWLOCRRC;;;SU)'
+            $WrongSecurityDescriptorSddl = 'D:(A;;CCLCSWRPWPDTLOCRRC;;;SY)(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;BB)(A;;CCLCSWLOCRRC;;;SU)'
             net user $userName $testPass /add > $null
             $password = ConvertTo-SecureString $testPass -AsPlainText -Force
             $creds = [pscredential]::new(".\$userName", $password)
 
+            $svcbinaryname = New-Item -Path TestDrive:\TestExecutable.exe -ItemType File
```

Comment:
I don't understand why this was changed.

Reply:
Because `Get-Command $svcbinaryname` fails as there is no `TestService` executable available. Will this be present in build agent ? Added this change when doing testing locally ?

Reply:
Yes, as I understand it, the TestService is part of the test set-up and you should revert this change.

@adityapatwardhan  can you confirm?